entrypoint.sh

```diff
@@ -13,46 +16,44 @@ REPOSITORY=${INPUT_REPOSITORY:-$GITHUB_REPOSITORY}
 set -e
 
 [ -z "${INPUT_GITHUB_TOKEN}" ] && {
-    echo 'Missing input "github_token: ${{ secrets.GITHUB_TOKEN }}".';
-    exit 1;
-};
+  echo 'Missing input "github_token: ${{ secrets.GITHUB_TOKEN }}".'
+  exit 1
+}
 
 echo "Repository: $REPOSITORY"
 echo "Actor: $GITHUB_ACTOR"
 
 echo "Installing requirements..."
-pip install commitizen $INPUT_EXTRA_REQUIREMENTS
+pip install "$INPUT_COMMITIZEN_VERSION" "$INPUT_EXTRA_REQUIREMENTS"
```

Comment:
I tested it locally, seems an issue comes from double quotes here (as default for this variable is `''`) !

```suggestion
pip install "$INPUT_COMMITIZEN_VERSION" $INPUT_EXTRA_REQUIREMENTS
```